content/file/file.go

```diff
@@ -104,6 +104,9 @@ type Store struct {
 	// manifest and config file, while leaving only named layer files.
 	// Default value: false.
 	IgnoreNoName bool
+	// AutoExtract controls if push operations should extract files automatically.
+	// Default value: true.
+	AutoExtract bool
```

Comment:
I think we should call this `AutoPack` or `SkipUnpack`(with default value: `false`), as the value of `AnnotationUnpack` is `io.deis.oras.content.unpack`.

Reply:
renamed the variable as `SkipUnpack`